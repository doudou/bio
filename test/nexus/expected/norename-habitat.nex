#NEXUS
[/home/cverna/.arb_tmp/tmp/gde19576_1 -- data title]

[Name: LorLacte          Len:  1159  Check: 2338B739]
[Name: LorLact5          Len:  1159  Check: 2338B739]
[Name: LorLact2          Len:  1159  Check:  580DE0F]
[Name: BtlSymb2          Len:  1159  Check: DFA31037]
[Name: LorLact3          Len:  1159  Check: 2338B739]

begin data;
 dimensions ntax=5 nchar=20;
 format datatype=rna interleave=yes missing=. gap=-;
  matrix
 LorLacte  GCGUGGAAUCUGCCUAGUAG
 LorLact5  GCGUGGAAUCUGCCUAGUAG
 LorLact2  GCGUGGAAUCUGCCUAGUAG
 BtlSymb2  GCGUGGAAUCUGCCUAGUAG
 LorLact3  GCGUGGAAUCUGCCUAGUAG
;
  end;
BEGIN TRAITS;
  Dimensions NTRAITS=2;
  Format labels=yes missing=? separator=Comma;
  TraitLabels hab0 hab1;
  Matrix
LorLacte 1,0
LorLact5 2,1
LorLact2 0,1
BtlSymb2 0,1
LorLact3 1,0
;
END;
