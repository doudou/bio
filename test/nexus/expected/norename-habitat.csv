name,hab0,hab1
LorLacte,1,0
LorLact5,2,1
LorLact2,0,1
BtlSymb2,0,1
LorLact3,1,0
